ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/StateView.hs

```diff
@@ -45,12 +46,14 @@ instance Condense ChainSyncException where
 -- mocked ChainSync server threads).
 data StateView blk = StateView {
     svSelectedChain       :: AnchoredFragment (Header blk),
-    svChainSyncExceptions :: [ChainSyncException]
+    svChainSyncExceptions :: [ChainSyncException],
+    svTipBlock            :: Maybe blk
```

Comment:
Yes, this deserves documentation indeed. I can answer the question, though: it can be the tip of the selection or the immutable tip or anything in between. It's the most recent point in the selection (incl. anchor) for which we have a block (body, not just header).

Reply:
Done in latest version